.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+<!--
+    Thank you for submitting the PR! We appreciate you spending the time to work on these changes.
+
+    Please help us understand your motivation by explaining why you decided to make this change.
+
+    Happy contributing!
+-->
+
+## Motivation
+
+(Write your motivation here)
+
+## Explanation of Changes
+
+<!-- Please explain why you made these changes the way you did.  -->
+
+(Write your explanation here)
+
+## Testing
+
+<!--
+    How do you test these changes?
+	What command do you run to test these changes specifically?
+-->
+
+(Write your test plan here)
+
+## Dependcy Chain
+[chain]: https://github.com/sedaprotocol/seda-chain
+[explorer]: https://github.com/sedaprotocol/seda-explorer
+[overlay-rs]: https://github.com/sedaprotocol/overlay-rs
+[overlay-ts]: https://github.com/sedaprotocol/seda-overlay-ts
+[sdk]: https://github.com/sedaprotocol/seda-sdk
+
+Think about the changes made in this PR and see if you need to make an issue/pr on the following repos.
+
+- [ ] [chain][chain]
+- [ ] [explorer/indexer][explorer]
+- [ ] [overlay-ts][overlay-ts]
+- [ ] [overlay-rs][overlay-rs]
+- [ ] [sdk][sdk]
+
+Should be consider if this pr changes:
+- events
+- gas limits
+- messages
+- signing/committing
+- protobuf
```

Comment:
Is this intended to be here?

Reply:
Maybe it's enough to add a couple of these as examples in parenthesis on line 35?

Reply:
I adjusted it to that.